documentation/modules/rn-2.2.adoc

```diff
@@ -49,3 +57,11 @@ This issue only affects the `Importer` pod log and warm migration. Cold migratio
 .Deleting migration plan does not remove temporary resources.
 
 Deleting a migration plan does not remove temporary resources such as `Importer` pods, `Conversion` pods, config maps, secrets, failed VMs and data volumes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[*BZ#2018974*]) You must archive a migration plan before deleting it in order to clean up the temporary resources.
+
+.Unclear error status message for VM with no operating system
+
+The error status message for a VM with no operating system on the *Migration plan details* page of the web console does not describe the reason for the failure. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[*BZ#2008846*])
+
+.Network, storage, and VM names in the Plan CR are not displayed in the web console.
```

Comment:
I just realized "names" gets a bit lost. Perhaps this is better: "Network, storage, and VMs referenced by name in the Plan CR are not displayed in the web console".